README-source.md

```diff
@@ -707,6 +706,21 @@ module.exports = {
 }
 ```
 
+### `bundle.targetUrl`
+
+> `bundle.targetUrl` is only available in the `performSubscribe` and `performUnsubscribe` methods for webhooks
+
+This the url to which you should send hook data. It'll look something like `https://hooks.zapier.com/1234/abcd`. We provide it so you can make some sort of POST request to your server and store this as a destination for new info.
```

Comment:
I'm fine with both `url` and `URL`, but we should be consistent across the entire document. So I think we should be using `URL` here.

Also, I'd suggest editing this:

> We provide it so you can make some sort of POST request to your server and store this as a destination for new info.

to:

> We provide it so we can make a POST request to your server, and your server should store this URL to hit when there's new data.